src/test/unit_tests/jest_tests/test_nc_nsfs_bucket_cli.test.js

```diff
@@ -511,7 +511,8 @@ describe('manage nsfs cli bucket flow', () => {
             await set_path_permissions_and_owner(bucket_defaults.path, account_defaults2, 0o700);
             await exec_manage_cli(TYPES.BUCKET, action, bucket_options);
             const bucket = await config_fs.get_bucket_by_name(bucket_defaults.name);
-            expect(bucket.bucket_owner).toBe(account_defaults2.name);
+            const owner_data = await config_fs.get_identity_by_id(bucket.owner_account);
```

Comment:
Why did you change it from "bucket_owner" to "owner_data"?

Reply:
because it's the owner data, bucket_owner IMO should be the bucket owner name as we use on bucketspaceFS

Reply:
We use the `bucket_owner` as a tern of the owner's name only, and `owner data` and the account (objects) which is the owner of the bucket?
I'm fine with it, I think a short comment can help.